--- a/Config.py	2020-06-16 20:09:37.105307516 +0200
+++ b/Config.py	2020-06-16 20:15:58.795527637 +0200
@@ -19,10 +19,10 @@
     try:
         head, tail = os.path.split(path_)
         while tail:
-            if tail == "LNP":
+            if tail == "announcement":
                 # Head is LNP install dir
                 for item in os.listdir(head):
-                    if re.match('[Dd]warf\s*[Ff]ortress\s*\d+\.\d+\.\d+', item):
+                    if re.match('[Dd]warf\s*[Ff]ortress\s*\d+\.\d+\.\d+', item) or re.match('df_\d+_\d+\_linux', item):
                         if os.path.isdir(os.path.join(head, item)):
                             path_ = os.path.join(head, item)
                             if os.path.isfile(os.path.join(path_, 'gamelog.txt')):
