--- dfhack-0.42.06-alpha2.orig/plugins/CMakeLists.txt	2016-02-14 02:02:45.789271706 +0100
+++ dfhack-0.42.06-alpha2/plugins/CMakeLists.txt	2016-02-14 02:08:24.096090552 +0100
@@ -171,6 +171,7 @@
     DFHACK_PLUGIN(workflow workflow.cpp LINK_LIBRARIES lua)
     DFHACK_PLUGIN(workNow workNow.cpp)
     DFHACK_PLUGIN(zone zone.cpp LINK_LIBRARIES lua)
+    add_subdirectory(twbt)
 endif()
 
 # this is the skeleton plugin. If you want to make your own, make a copy and then change it
