--- a/tkgui/tkgui.py	2020-06-22 23:04:12.000000000 +0200
+++ b/tkgui/tkgui.py	2022-01-05 23:04:59.465055122 +0100
@@ -559,7 +559,7 @@
             exc_info = sys.exc_info()
             messagebox.showerror(
                 title='Error launching Dwarf Fortress',
-                message=exc_info[1].message)
+                message=exc_info[1])
 
     def run_init(self):
         """Opens the init editor."""
