--- a/plugins/kloker.cpp	2020-06-01 15:11:31.000000000 +0200
+++ b/plugins/kloker.cpp	2021-04-05 18:16:07.421851575 +0200
@@ -44,6 +44,7 @@
 #include "df/nemesis_record.h"
 #include "df/historical_entity.h"
 #include "df/historical_figure.h"
+#include "df/historical_figure_relationships.h"
 #include "df/historical_figure_info.h"
 #include "df/histfig_hf_link.h"
 #include "df/histfig_hf_link_type.h"
