--- a/plugins/kloker.cpp	2020-06-01 15:11:31.000000000 +0200
+++ b/plugins/kloker.cpp	2021-04-17 13:58:53.751350483 +0200
@@ -45,6 +45,7 @@
 #include "df/historical_entity.h"
 #include "df/historical_figure.h"
 #include "df/historical_figure_info.h"
+#include "df/historical_figure_relationships.h"
 #include "df/histfig_hf_link.h"
 #include "df/histfig_hf_link_type.h"
 #include "df/activity_event.h"
@@ -6103,7 +6104,7 @@
 
     if (show_details != 0 && show_details < 4) {
 
-        if (ds.size() > to_lim) {
+        if ((ds.size() > to_lim) && (to_lim >= 3)) {
             ds.resize(to_lim - 3); ds += "...";
         }
         ds.resize(to_max);
@@ -6128,7 +6129,7 @@
         if (rn + dn > to_lim) {
             rg = " Rgds " + cur->regards;
             rn = rg.size();
-            if (rn + dn > to_lim) {
+            if ((rn + dn > to_lim) && (to_lim - rn >= 3)) {
                 ds.resize(to_lim - rn - 3);
                 ds += ".. ";
                 dn = ds.size();
@@ -6154,7 +6155,7 @@
             rg += ",Gods " + cur->godline;
         int rn = rg.size(), dn = ds.size();
 
-        if (rn + dn > to_lim) {
+        if ((rn + dn > to_lim) && (to_lim - rn >= 3)) {
             ds.resize(to_lim - rn - 3);
             ds += ".. ";
             dn = ds.size();
