--- a/dfhack-config/init/dfhack.init	2022-09-11 12:39:55.299101105 +0200
+++ b/dfhack-config/init/dfhack.init	2022-09-11 12:40:15.791492446 +0200
@@ -3,3 +3,13 @@
 #
 # You can extend or override DFHack's default configuration by adding commands
 # to this file.
+
+#######################################################
+# Loading TWBT Default multirender                    #
+#######################################################
+
+# Number of levels to render
+multilevel 6
+
+# making the shadow a little darker, better for a bright tileset.
+multilevel shadowcolor 0 0 0 0.5
