--- a/plugins/CMakeLists.txt	2020-10-11 15:32:14.661815957 +0200
+++ b/plugins/CMakeLists.txt	2020-10-11 15:32:24.381961312 +0200
@@ -177,6 +177,7 @@
     dfhack_plugin(workNow workNow.cpp)
     dfhack_plugin(xlsxreader xlsxreader.cpp LINK_LIBRARIES lua xlsxio_read_STATIC zip expat)
     dfhack_plugin(zone zone.cpp LINK_LIBRARIES lua)
+    add_subdirectory(twbt)
 endif()
 
 # this is the skeleton plugin. If you want to make your own, make a copy and then change it
