--- a/dfhack.init-example	2022-06-05 13:11:45.285701674 +0200
+++ b/dfhack.init-example	2022-06-05 13:12:35.067445005 +0200
@@ -294,3 +294,14 @@
 # Create a file named "onLoad.init" to run commands when a world is loaded
 # and/or create a file named "onMapLoad.init" to run commands when a map is
 # loaded. See the hack/examples/init/ directory for useful pre-made init files.
+
+#######################################################
+# Loading TWBT Default multirender                    #
+#######################################################
+
+# Number of levels to render
+multilevel 6
+
+# making the shadow a little darker, better for a bright tileset.
+multilevel shadowcolor 0 0 0 0.5
+
