--- lnp-0.10f.orig/core/launcher.py	2016-02-18 23:06:38.721190974 +0100
+++ lnp-0.10f/core/launcher.py	2016-02-18 23:09:16.625881759 +0100
@@ -133,6 +133,7 @@
                     sys._MEIPASS in environ['TK_LIBRARY']): # pylint:disable=no-member
                 del environ['TK_LIBRARY']
 
+        environ['LD_PRELOAD'] = '/usr/lib/libz.so'
         lnp.running[path] = subprocess.Popen(
             run_args, cwd=workdir, env=environ)
         return True
